src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,40 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+
+import './TodoInfo.scss';
```

Comment:
Consider organizing imports in alphabetical order for better readability and maintainability. 📚